crates/store/re_types_core/src/as_components.rs

```diff
@@ -207,10 +207,22 @@ mod tests {
     use itertools::Itertools as _;
     use similar_asserts::assert_eq;
 
+    use crate::{Component as _, ComponentDescriptor};
+
     #[derive(Clone, Copy, Debug, PartialEq, Eq, bytemuck::Pod, bytemuck::Zeroable)]
     #[repr(transparent)]
     pub struct MyColor(pub u32);
 
+    impl MyColor {
```

Comment:
bit annoying that we can't use `re_log_types::test_types::MyColor` here? 🤔 
(Not worth refactor I figure if it's even possible)